tests/phpunit/tests/kses.php

```diff
@@ -1955,8 +1955,10 @@ public function wp_kses_preserves_html_comments( $html_comment, $expected_output
 	 */
 	public static function data_html_containing_various_kinds_of_html_comments() {
 		return array(
-			'Normative HTML comment'            => array( 'before<!-- this is a comment -->after', 'before<!-- this is a comment -->after' ),
-			'Closing tag with invalid tag name' => array( 'before<//not a tag>after', 'before<//not a tag>after' ),
+			'Normative HTML comment'                     => array( 'before<!-- this is a comment -->after', 'before<!-- this is a comment -->after' ),
+			'Closing tag with invalid tag name'          => array( 'before<//not a tag>after', 'before<//not a tag>after' ),
+			'Incorrectly opened comment (Markup declaration)' => array( 'before<!also not a tag>after', 'before<!also not a tag>after' ),
+			'Incorrectly opened comment (Question mark)' => array( 'before<?also not a tag>after', 'before<?also not a tag>after' ),
```

Comment:
```suggestion
			'Normative HTML comment'                          => array( 'before<!-- this is a comment -->after', 'before<!-- this is a comment -->after' ),
			'Closing tag with invalid tag name'               => array( 'before<//not a tag>after', 'before<//not a tag>after' ),
			'Incorrectly opened comment (Markup declaration)' => array( 'before<!also not a tag>after', 'before<!also not a tag>after' ),
			'Incorrectly opened comment (Question mark)'      => array( 'before<?also not a tag>after', 'before<?also not a tag>after' ),
```

Arrow indentions.

Reply:
Thanks @mukeshpanchal27 for the suggestion, but I believe that the WPCS formatting rules prefer misalignment here. You can try changing them, and see it reject the PR.